server/monitor.go

```diff
@@ -277,6 +304,10 @@ func (s *Server) Connz(opts *ConnzOptions) (*Connz, error) {
 		// Fill in user if auth requested.
 		if auth {
 			ci.AuthorizedUser = client.opts.Username
+			// Add in account iff not the global account.
+			if client.acc != nil && (client.acc.Name != globalAccountName) {
```

Comment:
.. but here you check for `client.acc != nil`...

Reply:
Yes they can make it into closed without.

Reply:
ok